search_username/catalog/views.py

```diff
@@ -0,0 +1,47 @@
+from django.shortcuts import render, redirect
+
+from InstagramAPI import InstagramAPI
+
+user_name = 'case_iphone_lviv2019'
```

Comment:
It's better to move this data into the configuration file (e.g. settings.py, etc.)

Reply:
moved